src/index.d.ts

```diff
@@ -14,7 +14,7 @@ declare namespace preact {
 		type: ComponentType<P> | string | null;
```

Comment:
Maybe replace string with KeyOf(HTMLTypes) to increase safety

Reply:
Is it necessary to think about custom elements?

Reply:
Good point